LICENSE.md

```diff
@@ -136,3 +136,6 @@ associated software packages may are listed below:
 10. ds++/terminal.h and ds++/terminal_base.h developed by Ondřej Čertík
     https://github.com/certik/terminal
     MIT License
+
+11. experimental/mdspan - https://github.com/kokkos/mdspan
```

Comment:
It might be useful to to state the date this code was retrieved and/or the github branch/commit hash, in case future changes need to be brought in.

Reply:
Fixed in 8c4682e